.git2gus/config.json

```diff
@@ -1,4 +1,5 @@
 {
   "productTag": "a1aB0000000g309IAA",
-  "defaultBuild": "220"
+  "defaultBuild": "224",
+  "hideWorkItemUrl": true
```

Comment:
I usually dislike double negative configuration objects and variable names. just saying!

Reply:
agree! makes sense, i did it because the default is the other way around (to show the url) and there is other repositories already using it; that being said, i believe we can improve the api and achieve that other goal (all existing config keep behaving the same as is)

Reply:
we can do `showWorkItemURL: false` while it is true by default :)